runtime/doc/vim_diff.txt

```diff
@@ -38,7 +39,13 @@ these differences.
 - 'wildmode' defaults to "list:longest,full"
 
 ==============================================================================
-3. Changed features					 *nvim-features-changed*
+3. New keymaps        			   		          *nvim-keymaps*
+
+A small set of keymaps has been changed in nvim's default configuration. These
+are not hardcoded, and can be disabled by the user (see |nvim-extra-keymaps|).
```

Comment:
Not sure about the whole third-person thing, and I think this could be shortened a bit.
Maybe something like this?
```
However, these can be disabled: see |nvim-disable-crap|.
```